.github/workflows/lint.yml

```diff
@@ -0,0 +1,9 @@
+name: Lint Scripts
+on: [pull_request]
+jobs:
+  shellcheck:
+    name: Shellcheck
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: azohra/shell-linter@latest
```

Comment:
Could we run a pinned version instead of `latest`? What about https://github.com/azohra/shell-linter/releases/tag/v0.6.0? It might be worth considering that this repo is not built so often.

Reply:
Good point!